test/schedule_manager_test.exs

```diff
@@ -28,72 +29,67 @@ defmodule Verk.ScheduleManagerTest do
   test "handle_info :fetch_retryable without jobs to retry", %{ script: script } do
     state = %State{ redis: :redis }
     now = :now
-    expect(Timex.Time, :now, [:seconds], now)
     expect(Redix, :command, [:redis, ["EVALSHA", script, 1, "retry", now]], {:ok, nil})
 
     assert handle_info(:fetch_retryable, state) == { :noreply, state }
     assert_receive :fetch_retryable
 
-    assert validate [Timex.Time, Redix]
+    assert validate [ Redix]
   end
 
-  test "handle_info :fetch_retryable with jobs to retry", %{ script: script } do
+  test "handle_info :fetch_retryable with jobs to retry", %{ script: script  } do
     state = %State{ redis: :redis }
-    now = :now
+    now = Time.now
+    expect(Time, :now, [], now)
     encoded_job = "encoded_job"
-    expect(Timex.Time, :now, [:seconds], now)
-    expect(Redix, :command, [:redis, ["EVALSHA", script, 1, "retry", now]], {:ok, encoded_job})
+    expect(Redix, :command, [:redis, ["EVALSHA", script, 1, "retry", now], state], {:ok, encoded_job})
```

Comment:
I'll give that a shot—I'm pretty sure I added state there to try to get the tests passing, but it didn't work so it shouldn't have been committed.